linera-views/src/rocksdb.rs

```diff
@@ -171,6 +174,9 @@ impl KeyValueStoreClient for DbInternal {
 
 #[async_trait]
 impl KeyValueStoreClient for RocksdbClient {
+    // The maximum size of values in RocksDB is 3G
+    // That is 3221225472 and so for offset reason we decrease by 40
```

Comment:
How did you find 40? Does it need to be tracked as a constant? Should we take a larger security margin?

Reply:
It is purely arbitrary.
In truth, I have little confidence in the officially published constants. 3G sounds right.
400kB sounds not the kind of constants I would make my life depend.